torchrl/envs/libs/dm_control.py

```diff
@@ -272,6 +272,11 @@ class DMControlEnv(DMControlWrapper):
     """
 
     def __init__(self, env_name, task_name, **kwargs):
+        if not _has_dmc:
+            raise ImportError(
+                "dm_control python package was not found."
+                "Please install this dependency."
+            )
```

Comment:
I think it's better to check it with `_has_dmc` compared to `self.available_envs`, because it's easier to track why `_has_dmc` is `False`

Reply:
you're absolutely right!